odoo/tools/translate.py

```diff
@@ -1042,8 +1042,10 @@ def trans_load_data(cr, fileobj, fileformat, lang, lang_name=None, verbose=True,
             fields = ['type', 'name', 'res_id', 'src', 'value', 'comments']
 
             # Make a reader for the POT file and be somewhat defensive for the
-            # stable branch.
-            if fileobj.name.endswith('.po'):
+            # stable branch
```

Comment:
do not modify existing lines if not necessary ;)

Reply:
duly noted! ;-)